Planets and Systems/Harr'qac.md

```diff
@@ -36,52 +36,103 @@
+Harr'qaci Skrell usually build circular houses, which often have many entrances around the ground floor. The usual living house for one family has 4 floors, the ground floor being entrance (since doors are at least on four sides), which can be also used for less private meetings or celebrations. First floor usually houses the kitchen, living room and toilet. Second floor contains private rooms or offices and usually a toilet too. Third floor is mostly the same as second, but it doesn't usually have a toilet. In he middle of each floor, there is either an elevator or a circular staircase. On the top of the houses, there are balconies, in which some Skrellian families have little gardens. There's also a new trend currently spreading in Skrriol and Qrri'Mal, to build narrow metal bridges between balconies of houses with simillar height, if the families are like meeting each other.
 
 ## Government
 
-Similliarly to Qerr'Balak, there are many city-states governing themselves. However, when colonising the planet, Skrell chose to try a similliar yet new form of government. City-states on each continent are grouped together. Qerr-Skria of these city-states elect a representative of their continent, called Qerr. This way, cities manage themselves, they decide about continent-wide things together and are represented by one Skrell for each continent when discussing global decisions. The three Qerrs meet in Government District of the University of Harr'qac.
+Similliarly to Qerrbalak, there are many city-states governing themselves. However, when colonising the planet, Skrell chose to try a similliar, yet new form of government. City-states on each continent are grouped together. Qerr-Skria of these city-states elect a representative of their continent, called Qerr-Kirri. This way, cities manage themselves, they decide about continent-wide things together and are represented by one Skrell for each continent when discussing global decisions. The three Qerr-Kirris meet in Government District of the University of Harr'qac and are very respected amongst the population.
 
 ## Education
 
-The most famous academy of Harr'qac is The University of Harr'qac. It's not located any any city, but it rather creates a city like any other itself. It's located on the island of Korm. The university takes almost whole island, with lots of parks, services, leisure areas... It is supported by other cities, as the global government meets there.
+The most famous academy of Harr'qac is The University of Harr'qac. It's not located in any city, but it rather creates a city like any other itself. It's located on the island of Korm. The university takes almost whole island, with lots of parks, services, leisure areas and the like. It is supported by other cities, as the global government meets there.
```

Comment:
How about this:
It was not originally located within any city, but one was built up around it to facilitate its needs.

Although, the city part also includes the various areas and building used for education, not only the services and facilities for living.